common/djangoapps/django_comment_common/signals.py

```diff
@@ -0,0 +1,12 @@
+"""Signals related to the comments service."""
+
+from django.dispatch import Signal
+
+
+thread_created = Signal(providing_args=['user'])
```

Comment:
Note: if used in the future, these signals will most likely need to be extended to include more information such as the thread ID. I debated adding that information right now, but decided against it. It seems pointless to speculatively add arguments without knowing if or why they'll be used. The only thing that's needed for teams is the user, and more information can be added without needing to modify the existing code.